engines/query-sparql/test/QuerySparql-test.ts

```diff
@@ -421,6 +421,80 @@ describe('System test: QuerySparql', () => {
             .toThrow('Illegal simultaneous usage of extensionFunctionCreator and extensionFunctions in context');
         });
 
+        describe('filter pushdown behaviour with extension functions', () => {
+          let containsFilter: boolean;
```

Comment:
Could you add a small word on what this test does?

Just the general how of how you pass a fetch and check whether a filter is contained from within that fetch.